landa/water_body_management/doctype/fish_species/fish_species.json

```
@@ -63,6 +63,11 @@
    "fieldtype": "Data",
    "label": "Minimum Size"
   },
+  {
+    "fieldname": "typical_weight",
```

```suggestion
    "fieldname": "typical_weight_in_kg",
```

Die Maßeinheit ist auch im Code wichtig zu wissen, daher sollte sie auch im `fieldname` vorkommen.